ui/src/views/WorkerManager/WMViewWorkerTypes/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { hot } from 'react-hot-loader';
+import React, { Component } from 'react';
+import { graphql } from 'react-apollo';
+import Spinner from '@mozilla-frontend-infra/components/Spinner';
+import Dashboard from '../../../components/Dashboard';
+import workerTypesQuery from './WMWorkerTypes.graphql';
+import ErrorPanel from '../../../components/ErrorPanel';
+import WMWorkerTypesTable from '../../../components/WMWorkerTypesTable';
+import Search from '../../../components/Search';
+
+@hot(module)
+@graphql(workerTypesQuery)
+export default class WMWorkerTypesView extends Component {
+  state = {
+    workerTypeSearch: '',
+  };
+
+  handleWorkerTypeSearchSubmit = workerTypeSearch => {
+    this.setState({ workerTypeSearch });
+  };
+
+  render() {
+    const {
+      data: { loading, error, WMWorkerTypeSummaries },
+      match: { path },
+    } = this.props;
+    const { workerTypeSearch } = this.state;
+
+    return (
+      <Dashboard
+        title="Worker Types"
+        search={
+          <Search
+            disabled={loading}
+            onSubmit={this.handleWorkerTypeSearchSubmit}
+            placeholder="Worker type contains"
+          />
+        }>
+        <>
```

Comment:
Use `<Fragment>` instead for consistency in the codebase.

Reply:
Can you link me to the documentation listing the things you want to be consistent in this codebase? (If there is none, it's up to me then 😛 )

Reply:
There are two ways to return multiple elements without adding extra noes to the DOM:
* `<>`
* `Fragment`

In tc-ui, we've only used `Fragment`. For consistency, I was suggesting we keep using the `Fragment` component. If `<>` makes things easier for you, then sure, let's keep it.